pytest_pyodide/hook.py

```diff
@@ -89,3 +97,16 @@ def pytest_pycollect_makemodule(module_path: Path, path: Any, parent: Any) -> No
     rewrite_asserts(tree2, source, strfn, REWRITE_CONFIG)
     REWRITTEN_MODULE_ASTS[strfn] = tree2
     orig_pytest_pycollect_makemodule(module_path, parent)
+
+
+def pytest_generate_tests(metafunc: Any) -> None:
+    runtime = metafunc.config.option.runtime
+
+    if runtime == "all":
+        runtime = RUNTIMES
+
+    if "runtime" in metafunc.fixturenames:
+        if runtime == "host":
+            metafunc.parametrize("conf", pytest.skip("Non-host test"))
```

Comment:
What's conf? Shouldn't it be runtime?

Reply:
Actually, that was the wrong way to skip a test (It was skipping an entire module)...